packages/lesswrong/server/analyticsWriter.ts

```diff
@@ -98,7 +99,10 @@ async function writeEventsToAnalyticsDB(events: {type, timestamp, props}[]) {
       
       inFlightRequestCounter.inFlightRequests++;
       try {
-        await connection.none(query);
+        await Promise.all([
+          connection.none(query),
```

Comment:
It looks like both `getAnalyticsConnection` and `getMirrorAnalyticsConnection` return `AnalyticsConnectionPool | null`, so we should probably either have `connection?.none(query)` checking for null (or `connection!.none(query)` if we're _really_ sure) or change the return type of `getAnalyticsConnection` to just `AnalyticsConnectionPool` if we can guaratee that.

Reply:
Yeah, `connection?.none(query)` makes sense